### ISSUE TITLE
Shortcuts on menus, button tooltips, Keyboard shortcut help don't update

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5685]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : all versions
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- Customize a keyboard shortcut
- Look at that command on the main menu, hover over its button elsewhere (toolbar)
- Look at command on Help / Keyboard Shortcuts Help

### Describe the problem in detail
- Shortcut shown is the original, not the customized one

### Describe the behavior you expected
- Should be showing the currently active shortcut, not the default

**Comment by gtritchie** [comment]
Marking as accessibility issue since this is confusing for keyboard users who make customizations and rely on these cues to recall their changes.

**Comment by gtritchie** [comment]
Resolving as a duplicate, this is already tracked, essentially, by both #5256 and #5595.
